src/nih_wayfinding/app/scripts/views/profile/new-profile/new-profile-controller.js

```diff
@@ -38,37 +37,29 @@
         /**
          * Set private var for whether using a wheelchair or scooter.
         */
-        function setUsingWheelchair(amUsing) {
-            ctl.newUser.setPreference('wheelchairRequired', amUsing.value);
+        function setUsingAssistance(amUsing) {
+            ctl.newUser.setPreference('assistanceRequired', amUsing.value);
             if (amUsing.value) {
-                ctl.wheelchairTypeDialog = true;
+                ctl.assistanceTypeDialog = true;
             } else {
-                ctl.setWheelchairType({ value: null });
+                ctl.setAssistanceType({ value: null });
             }
         }
 
         /**
          * Set private var for whether using a wheelchair or scooter.
         */
-        function setWheelchairType(chairType) {
-            ctl.newUser.setPreference('wheelchairPowered', chairType.value);
+        function setAssistanceType(assistanceType) {
+            ctl.newUser.setPreference('assistanceType', assistanceType.value);
             setStep(3);
         }
 
         /**
-         * Set private var for profile challenge level.
-         */
-        function setChallengeLevel(challenge) {
-            ctl.newUser.setPreference('challenge', challenge.value);
-            setStep(4);
-        }
-
-        /**
          * Set private var for profile steepTerrainComfort level.
          */
         function setSteepTerrainComfort(comfort) {
             ctl.newUser.setPreference('steepTerrainComfort', comfort.value);
-            setStep(5);
+            setStep(4);
```

Comment:
We may want to make some sort of enum object for these if the flow ever gets more complex.